roles/neutron-common/tasks/main.yml

```diff
@@ -61,6 +61,18 @@
   notify:
      - restart neutron services
 
+- name: neutron lbaas config
+  template: src=etc/neutron/services/loadbalancer/haproxy/lbaas_agent.ini dest=/etc/neutron/services/loadbalancer/haproxy mode=0644
+  notify:
+     - restart neutron services
+  when: neutron.lbaas.enable
+
+- name: update-alternatives for lbaas
+  shell: update-alternatives --install /usr/local/bin/{{ item }} {{ item }} /opt/openstack/current/neutron/bin/{{ item }} 10 --force
+  with_items:
+    - neutron-lbaasv2-agent
+  when: neutron.lbaas.enable and openstack_install_method == 'package'
+  
```

Comment:
The config and service should go into the neutron-data (if these need to run on compute nodes) or neutron-data-network (if they just run in the same place the router and dhcp agents go). They should not be installed in the common role.